vmdb/app/models/configuration_profile.rb

```diff
@@ -1,4 +1,9 @@
 class ConfigurationProfile < ActiveRecord::Base
   include NewWithTypeStiMixin
+  include ReportableMixin
+
+  acts_as_miq_taggable
+
   belongs_to :configuration_manager
+  has_one    :provider, :through => :configuration_manager
```

Comment:
Have you tried what this does in reporting?  I can't remember if `has_one :through` works like `belongs_to`.  I thought we ignore or do some other special thing with belongs_to in reporting because you can create circular reports.